specification/metadata/schema/column_definition/columnname.md

```diff
@@ -12,6 +12,10 @@ ColumnName
 
 Column Name
 
+## Requirements
```

Comment:
Use this format in stead: 

## Content constraints

| Constraint      | Value          |
| :-------------- | :------------- |
| Column type     | Dimension      |
| Feature level   | Conditional    |
| Allows nulls    | True           |
| Data type       | String         |
| Value format    | Allowed Values |